src/components/Header/DrawLastUpdated.js

```diff
@@ -36,36 +36,36 @@ const drawLastUpdated = (lastUpdatedString, lang) => {
     display.textContent = lastUpdatedString;
     return;
   }
-  const relativeTime = {
-    en: formatDistanceToNow(lastUpdated, {
-      locale: enUS,
-      addSuffix: true,
-    }),
-    ja: formatDistanceToNow(lastUpdated, { locale: ja, addSuffix: true }),
-    es: formatDistanceToNow(lastUpdated, { locale: es, addSuffix: true }),
-  };
 
-  display.textContent = relativeTime[lang];
+  for (const language of LANGUAGES) {
+    addRelativeTimeLocalization(lastUpdated, language);
+  }
+
   display.setAttribute("title", lastUpdatedString);
-  i18next.addResource(
-    "en",
-    "translation",
-    "last-updated-time",
-    relativeTime["en"]
-  );
-  i18next.addResource(
-    "ja",
+  display.setAttribute("data-i18n", "last-updated-time");
+  display.textContent = i18next.getResource(
+    currentLanguage,
     "translation",
-    "last-updated-time",
-    relativeTime["ja"]
+    "last-updated-time"
   );
+};
+
+function addRelativeTimeLocalization(lastUpdated, language) {
```

Comment:
Minor request. We're using the following function declaration syntax in most places:

const fn = (args) => {
}

Would you mind switching it from function name(args) { } to that format?

Reply:
Done here: https://github.com/reustle/covid19japan/pull/312/commits/98e3175c02046b90f822527b23a61b93d2843e93

`const` declarations aren't hoisted so I had to move those to the top of the file.